### DISCUSSION TITLE
Possibility of adding a custom Delta wiki

### DISCUSSION THREAD

**Discussion by TheGiraffe3** [endless-sky/Endless-Sky-Delta#138] [General]
The current Delta wiki has links pointing to upstream's wiki. It might be possible to update our workflows so that they add upstream's wiki into Delta's wiki.
Or we could fork the endless-sky-wiki repository in a different account and have Delta pull from that.

Alternatively, we could just leave it as-is, but I think if we get the same specifications as upstream and those will be updated as upstream's, but we can also edit it, creating Delta content will be a lot easier.

Does look like the workflow might overwrite all the pages, which could be trouble.

**Comment by Zitchas** [comment]
Probably the most thorough methodology would be to fork the upstream wiki repository, and make changes to that fork. Then upstream changes can be synced into it in the same manner as we sync changes into Delta itself.

**Comment by Zitchas** [comment]
Alternatively, might be able to discuss having a main and Delta branch of that wiki. Main is upstream, Delta branch is for Delta, and then the workflow to pull into the wiki section here just needs the slight tweak to point at the branch instead.

**Reply by TheGiraffe3** [reply]
I think this might be the best approach; the problem would be only a few people have access to update the Delta branch.
Maybe a branch protection rule where...

**Comment by TheGiraffe3** [comment]
Well, I've got my fork set up to receive wiki updates, and it seems to work pretty well. The next step is testing whether it overwrites pages or just adds to them.
https://github.com/TheGiraffe3/endless-sky/wiki

**Comment by TheGiraffe3** [comment]
Alright; tested.
Turns out that it overwrites all files that are in the repository _but_ it doesn't delete pages that it doesn't recognize. So we could have a "Delta syntax changes" page... or just not bother.

Yeah, probably the best approach is having a "Delta" branch of the wiki repository, but that could run into problems with other people, as I mentioned earlier.
